php/Blockonomics.php

```diff
@@ -435,47 +447,11 @@ public function is_error_template($template_name) {
 
     // Adds the header to the blockonomics page
     public function load_blockonomics_header($template_name, $additional_script=NULL){
-        
-        // Lite mode will render without wordpress theme headers
-        if($this->is_lite_mode_active()){
-        ?>
-            <link rel="stylesheet" type="text/css" href="<?php echo plugins_url('css/order.css', dirname(__FILE__));?>">
-        <?php
-            if ($template_name === 'checkout') {
-        ?>
-            <script src="<?php echo plugins_url('js/vendors/reconnecting-websocket.min.js', dirname(__FILE__));?>" defer="defer"></script>
-            <script src="<?php echo plugins_url('js/vendors/qrious.min.js', dirname(__FILE__));?>" defer="defer"></script>
-            <script><?php echo $additional_script; ?></script>
-            <script src="<?php echo plugins_url('js/checkout.js', dirname(__FILE__));?>" defer="defer"></script>
-        <?php
-            }
-        } else {
-            add_action('wp_enqueue_scripts', 'bnomics_enqueue_stylesheets' );
-            
-            // wp_enqueue_scripts needs to be called before get_header(), but the scripts are loaded in footer as
-            // $in_footer is set to TRUE for scripts in bnomics_enqueu_scripts
-
-            if ($template_name === 'checkout') {
-                
-                add_action('wp_enqueue_scripts', 'bnomics_enqueue_scripts' );
-                
-                if (isset($additional_script)) {
-                    add_action('wp_enqueue_scripts', function () use ($additional_script) {
-                        wp_add_inline_script('bnomics-checkout', $additional_script, 'before');
-                    });
-                }
-            }
-
-            get_header();
-        }
-    }
-
-    // Adds the footer to the blockonomics page
-    public function load_blockonomics_footer($template_name){
-        
-        // Lite mode will render without wordpress theme footers
-        if(!$this->is_lite_mode_active()){
-            get_footer();
+        wp_enqueue_style( 'bnomics-style' );
+        if ($template_name === 'checkout') {
+            wp_add_inline_script('bnomics-checkout', $additional_script, 'before');
+            wp_enqueue_script( 'bnomics-checkout' );
+            wp_enqueue_script( 'copytoclipboard' );
```

Comment:
copytoclipboard should be defined as a dependency for bnomics-checkout.
```
wp_register_script( 'bnomics-checkout', plugins_url('js/checkout.js#deferload', __FILE__), array('reconnecting-websocket', 'qrious', 'copytoclipboard'), get_plugin_data( __FILE__ )['Version'] );
```
This allows us to simply use `wp_enqueue_script('bnomics-checkout');` to load all the scripts required

Reply:
done